frontend/src/old-pages/Configure/Queues/Queues.tsx

```diff
@@ -402,16 +407,33 @@ function Queue({index}: any) {
         </Box>
         <ColumnLayout columns={2}>
           <FormField
-            label={t('wizard.queues.subnet.label')}
+            label={
+              isMultiAZActive
+                ? t('wizard.queues.subnet.label.multiple')
+                : t('wizard.queues.subnet.label.single')
+            }
             errorText={subnetError}
           >
-            <SubnetSelect
-              value={subnetValue}
-              onChange={(subnetId: any) => {
-                setState(subnetPath, [subnetId])
-                queueValidate(index)
-              }}
-            />
+            {isMultiAZActive ? (
+              <SubnetMultiSelect
+                value={subnetsList}
+                onChange={({detail}: any) => {
```

Comment:
todo: avoid using inline callbacks in React render cycles, instead extract this block into a separate function wrapped in a `useCallback`

Reply:
Thanks, will fix in rev2